jetty-client/src/test/java/org/eclipse/jetty/client/HttpClientURITest.java

```diff
@@ -63,6 +63,13 @@ public void testIPv6Host() throws Exception
         Assert.assertEquals(HttpStatus.OK_200, request.send().getStatus());
     }
 
+    @Test(expected = IllegalArgumentException.class)
+    public void testIdnHost() throws Exception
```

Comment:
Please rename test method to be `testIDNHost()`, so that it's clear that it's an acronym and not a typo.